modules/wallet/scan.go

```diff
@@ -1,15 +1,20 @@
 package wallet
 
 import (
+	"errors"
 	"fmt"
 
 	"github.com/NebulousLabs/Sia/build"
+	"github.com/NebulousLabs/Sia/crypto"
 	"github.com/NebulousLabs/Sia/modules"
 	"github.com/NebulousLabs/Sia/types"
+	"github.com/NebulousLabs/bolt"
 )
 
 const scanMultiplier = 4 // how many more keys to generate after each scan iteration
 
+const outputSize = 350 // approx. size in bytes of an output and accompanying signature
```

Comment:
More precise testing shows that the value is closer to 200 bytes, with a small amount of variance (1-3 bytes) depending on the value of the output. 250 is probably a safe value.